src/grafana-custom/provisioning/datasources/mqtt-datasource.yaml

```diff
@@ -13,7 +13,7 @@ datasources:
       # Required by the plugin
       protocol: "tcp"
       
-      server: tcp://mqtt-broker_1:1883
+      server: tcp://mqtt-broker_2:1883
```

Comment:
this shouldn't change - your grafana instance is actually using a different mqtt-datasource.yml file